modules/nw-ovn-kubernetes-logs-cli.adoc

```diff
@@ -36,12 +36,12 @@ For example:
 +
 [source,terminal]
 ----
-$ oc logs ovnkube-master-7h4q7 -n openshift-ovn-kubernetes
+$ oc logs ovnkube-node-5dx44 -n openshift-ovn-kubernetes
 ----
 +
 [source,terminal]
 ----
-$ oc logs -f ovnkube-master-7h4q7 -n openshift-ovn-kubernetes -c ovn-dbchecker
+$ oc logs -f ovnkube-node-5dx44 -n openshift-ovn-kubernetes -c ovn-dbchecker
```

Comment:
I get the following error when I run this command on a 4.14 cluster: `error: container ovn-dbchecker is not valid for pod ovnkube-node-6m8fc` @tssurya and @anuragthehatter